cmd/utils/dev.go

```diff
@@ -151,19 +151,27 @@ func GetDevFromManifest(manifest *model.Manifest, devName string) (*model.Dev, e
 	} else if len(manifest.Dev) == 1 {
 		for name, dev := range manifest.Dev {
 			if devName != "" && devName != name {
-				return nil, fmt.Errorf("dev '%s' doesn't exist", devName)
+				return nil, oktetoErrors.UserError{
```

Comment:
We should be able to add a test that we are getting the correct error content. This seems important to test as it's part of the UI and very helpful to have.

Reply:
I realised that we didn't have any test for this function so I added test not just for this PR but for all the cases that don't require the user to add an input